addons/l10n_ar_withholding/data/template/account.tax-ar_ri.csv

```diff
@@ -0,0 +1,265 @@
+,,,,,,,,,,,base,invoice,,,,
+,,,,,,,,,,,tax,invoice,ri_retencion_iibb_se_aplicada,,,
+,,,,,,,,,,,base,refund,,,,
+,,,,,,,,,,,tax,refund,ri_retencion_iibb_se_aplicada,,,
+ri_tax_withholding_iibb_tn_applied,IIBB WTH Tucumán A,IIBB withholding Tucumán applied,True,4,fixed,0,tax_group_withholding_iibb_tn,none,supplier,untaxed_amount,,,,Ret IIBB Tucumán A,Retención IIBB Tucumán Aplicada,ri_seq_withholding_iibb_tn_applied
+,,,,,,,,,,,base,invoice,,,,
+,,,,,,,,,,,tax,invoice,ri_retencion_iibb_tn_aplicada,,,
+,,,,,,,,,,,base,refund,,,,
+,,,,,,,,,,,tax,refund,ri_retencion_iibb_tn_aplicada,,,
+ri_tax_withholding_iibb_ha_applied,IIBB WTH Chaco A,IIBB withholding Chaco applied,True,4,fixed,0,tax_group_withholding_iibb_ha,none,supplier,untaxed_amount,,,,Ret IIBB Chaco A,Retención IIBB Chaco Aplicada,ri_seq_withholding_iibb_ha_applied
+,,,,,,,,,,,base,invoice,,,,
+,,,,,,,,,,,tax,invoice,ri_retencion_iibb_ha_aplicada,,,
+,,,,,,,,,,,base,refund,,,,
+,,,,,,,,,,,tax,refund,ri_retencion_iibb_ha_aplicada,,,
+ri_tax_withholding_iibb_ct_applied,IIBB WTH Chubut A,IIBB withholding Chubut applied,True,4,fixed,0,tax_group_withholding_iibb_ct,none,supplier,untaxed_amount,,,,Ret IIBB Chubut A,Retención IIBB Chubut Aplicada,ri_seq_withholding_iibb_ct_applied
+,,,,,,,,,,,base,invoice,,,,
+,,,,,,,,,,,tax,invoice,ri_retencion_iibb_ct_aplicada,,,
+,,,,,,,,,,,base,refund,,,,
+,,,,,,,,,,,tax,refund,ri_retencion_iibb_ct_aplicada,,,
+ri_tax_withholding_iibb_fo_applied,IIBB WTH Formosa A,IIBB withholding Formosa applied,True,4,fixed,0,tax_group_withholding_iibb_fo,none,supplier,untaxed_amount,,,,Ret IIBB Formosa A,Retención IIBB Formosa Aplicada,ri_seq_withholding_iibb_fo_applied
+,,,,,,,,,,,base,invoice,,,,
+,,,,,,,,,,,tax,invoice,ri_retencion_iibb_fo_aplicada,,,
+,,,,,,,,,,,base,refund,,,,
+,,,,,,,,,,,tax,refund,ri_retencion_iibb_fo_aplicada,,,
+ri_tax_withholding_iibb_mi_applied,IIBB WTH Misiones A,IIBB withholding Misiones applied,True,4,fixed,0,tax_group_withholding_iibb_mi,none,supplier,untaxed_amount,,,,Ret IIBB Misiones A,Retención IIBB Misiones Aplicada,ri_seq_withholding_iibb_mi_applied
+,,,,,,,,,,,base,invoice,,,,
+,,,,,,,,,,,tax,invoice,ri_retencion_iibb_mi_aplicada,,,
+,,,,,,,,,,,base,refund,,,,
+,,,,,,,,,,,tax,refund,ri_retencion_iibb_mi_aplicada,,,
+ri_tax_withholding_iibb_ne_applied,IIBB WTH Neuquén A,IIBB withholding Neuquén applied,True,4,fixed,0,tax_group_withholding_iibb_ne,none,supplier,untaxed_amount,,,,Ret IIBB Neuquén A,Retención IIBB Neuquén Aplicada,ri_seq_withholding_iibb_ne_applied
+,,,,,,,,,,,base,invoice,,,,
+,,,,,,,,,,,tax,invoice,ri_retencion_iibb_ne_aplicada,,,
+,,,,,,,,,,,base,refund,,,,
+,,,,,,,,,,,tax,refund,ri_retencion_iibb_ne_aplicada,,,
+ri_tax_withholding_iibb_lp_applied,IIBB WTH La Pampa A,IIBB withholding La Pampa applied,True,4,fixed,0,tax_group_withholding_iibb_lp,none,supplier,untaxed_amount,,,,Ret IIBB La Pampa A,Retención IIBB La Pampa Aplicada,ri_seq_withholding_iibb_lp_applied
```

Comment:
better to have percent here as most are percentage

Reply:
Fixed